packages/nodes-base/nodes/Mattermost/v1/actions/channel/del/execute.ts

```diff
@@ -0,0 +1,25 @@
+import {
+	IExecuteFunctions,
+} from 'n8n-core';
+
+import {
+	IDataObject,
+	INodeExecutionData,
+} from 'n8n-workflow';
+
+import {
+	apiRequest,
+} from '../../../transport';
+
+export async function del(this: IExecuteFunctions, index: number): Promise<INodeExecutionData[]> {
```

Comment:
Why is this function and the folder called "del" and not "delete"?

Reply:
We named it `del` because `delete` is used for the [delete operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/delete) and was causing build problems.